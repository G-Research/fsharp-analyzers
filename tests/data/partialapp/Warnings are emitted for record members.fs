module M

let myFunc x y = x + y

type MyRec =
    {
        Field1: int
        Field2: int -> int
    }

    static member MyRecMember x = myFunc 23 x // should not warn
    static member MyRecMember2 x = myFunc x // should warn
