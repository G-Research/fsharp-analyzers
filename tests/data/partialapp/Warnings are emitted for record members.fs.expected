GRA-PARTAPP-001 | Warning | (12,35 - 12,41) | partial application should not be used: myFunc at (12,35--12,41)
